### ISSUE TITLE
Poor sound quality

### ISSUE THREAD

**Issue by MKSMV** [google/ExoPlayer#4378]
### Issue description
Poor sound quality with the Exoplayer

### Reproduction steps
the issue can be reproduced, using the ExoPlayer demo app.

### Link to test content
https://files.fm/u/7c4sskwq

### Version of ExoPlayer being used
2.8.1

### Device(s) and version(s) of Android being used
Xiaomi redmi 3, xiaomi MI A1, samsung j7,
Huawei Honor 4C

**Comment by AquilesCanta** [comment]
Hi @IvanIvanov3, after some digging around we couldn't conclude the issue was related to ExoPlayer. Perhaps you can help with the diagnosis. I was able to reproduce the issue on a linux desktop, but not on MacOs.

How was the media produced? Anything might be worth mentioning?

Considering the name of the file, I assume it was produced on iPhone. It's possible that ios/macos do special handling of the audio format. Right now, deeper research on this is low priority for us, since it is highly likely the issue lies in the media. If you can point us in the right direction to fix this, please do. In the meantime, we probably depend on external contributions.

**Comment by MKSMV** [comment]
Hi, **@AquilesCanta**, thanks for your attention.
The video was recorded on an iPhone with a microphone.

I tried to play the video with the MediaPlayer and the sound remained as bad as before. Then I added an FFmpeg extension for AAC decoding - the sound became better, but not perfect. What's strange, the sound is perfect in Windows.

Most likely, the reason of bad sound is not the Exo. If you decide to continue further research, I can find out more specific details from our ios developers.
